ide/coqide_main.mli

```diff
@@ -0,0 +1,17 @@
+(************************************************************************)
+(*  v      *   The Coq Proof Assistant  /  The Coq Development Team     *)
+(* <O___,, *   INRIA - CNRS - LIX - LRI - PPS - Copyright 1999-2017     *)
+(*   \VV/  **************************************************************)
+(*    //   *      This file is distributed under the terms of the       *)
+(*         *       GNU Lesser General Public License Version 2.1        *)
+(************************************************************************)
+
+(* This file avoids a race condition that occurs when compiling a .ml file
+   that does not have a corresponding .mli file *)
+
+(* These two functions are used only on Win32 platforms. Conditional compilation
+   triggers a 32 warning on other platforms, and we cannot disable the warning due
+   to the lack of attributes in camlp5. So we export them, although they are not
+   really used outside. *)
```

Comment:
Instead, we could either
- put the function definitions inside the IFDEF too. This would mean they're not checked by the compiler on other platforms, but with appveyor etc that should be OK
- or put `let _ = set_win32_path (* silence warning *)` (and the other function ofc) in the ELSE branch

Not sure which of these or keeping them in the mli is better, pick what you prefer.